### PR TITLE
Fix install entrypoints (fixes #499).

### LINKED ISSUE #499: Regression in `install_cmdstan` from 0.9.77 to develop
#### Summary:

Running `install_cmdstan` from the `develop` branch raises 

```python
Traceback (most recent call last):
  File "/usr/local/bin/install_cmdstan", line 8, in <module>
    sys.exit(main())
TypeError: main() missing 1 required positional argument: 'args'
```

#### Description:

Running `install_cmdstan` works as expected for `0.9.77`.

```bash
$ docker run --rm -it python:3.8 bash -c 'pip install cmdstanpy==0.9.77 && install_cmdstan -h'
[help message]
```

But it fails for the `develop` branch (and similarly for the `1.0.0rc2` on pypi).

```bash
$ docker run --rm -it python:3.8 bash -c 'pip install git+https://github.com/stan-dev/cmdstanpy && install_cmdstan -h'
[TypeError, see details above]
```

**Edit**: fixed a few typos.

### PR DESCRIPTION
**Pull request by tillahoffmann**
#### Submission Checklist

- [x] Run unit tests
- [x] Declare copyright holder and open-source license: see below

#### Summary

This PR fixes `setup.py` console script entrypoints as discussed in #499.

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Imperial College London

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -79,11 +79,15 @@ jobs:
       - name: Install CmdStan (Linux, macOS)
         if: matrix.os != 'windows-latest'
         run: |
+          install_cmdstan -h
```
Comment (tillahoffmann) [inline .github/workflows/main.yml:82]: Running the help message here to ensure the entrypoint works.

**Comment by tillahoffmann** [comment]
Accidentally force-pushed an empty commit which requires an additional approval to run the tests. History should be clean now.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -79,11 +79,15 @@ jobs:
...
+          install_cxx_toolchain -h
```
Comment (WardBrian) [inline .github/workflows/main.yml:83]: I believe this will fail on non-Windows

Comment (WardBrian) [reply .github/workflows/main.yml:83]: Seems like it doesn't, so we can leave it

**Review by WardBrian — APPROVED** [review]

**Comment by WardBrian** [comment]
Thanks!

**Pull request merged**